charts/datastore/templates/service.yaml

```diff
@@ -26,7 +26,11 @@ spec:
       name: {{ printf "%s-https" (include "datastore.name" .) }}
     {{- else }}
     - protocol: TCP
+      {{- if .Values.datastore.ingressPort }}
+      port: {{ .Values.datastore.ingressPort }}
+      {{- else }}
```

Comment:
have you tested it? 
I think you need to add another -protocol, port,etc..
https://stackoverflow.com/questions/45147664/how-to-expose-multiple-port-using-a-load-balancer-services-in-kubernetes

Reply:
![image](https://user-images.githubusercontent.com/69419999/108058230-66dd1880-705c-11eb-85a2-0c8ff462e456.png)

Reply:
this example taken from AWS which shows how to support both HTTP and HTTPS in the ingress.

in our case we need only one port which will serve for HTTPS in the external port and will forward plain HTTP to the container (via targetPort)

Reply:
The example you sent support my comment, You need to add instead of just port
    - protocol: TCP
      port: {{ .Values.datastore.ingressPort }}
      name: "additional-port"
      targetPort: {{ default 8080 .Values.datastore.dopContainerPort }}

Reply:
in your current code it will yield 

```
    - protocol: TCP
      port: 1234
      port: 80
      name: "additional-port"
      targetPort: {{ default 8080 .Values.datastore.dopContainerPort }}
```

which wont work